synapse/federation/federation_client.py

```diff
@@ -1335,15 +1335,15 @@ def from_json_dict(cls, d: JsonDict) -> "FederationSpaceSummaryEventResult":
         if any(not isinstance(v, str) for v in via):
             raise ValueError("Invalid event: 'via' must be a list of strings")
 
-        return cls(event_type, state_key, via, d)
+        return cls(event_type, room_id, state_key, via, d)
```

Comment:
this has ended up in the wrong commit, but nm

Reply:
Good shout, sorry about that. I had much rebasing in these branches... 😢